src/lib/layouts/hebrew.ts

```diff
@@ -7,11 +7,11 @@ import { LayoutItem } from "../interfaces";
 export default <LayoutItem>{
   layout: {
     default: [
-      "\u05e5 1 2 3 4 5 6 7 8 9 0 - = {bksp}",
-      "{tab} \u05e3 \u05df \u05e7 \u05e8 \u05d0 \u05d8 \u05d5 \u05ea \u05dd \u05e4 ] [ \\",
-      "{lock} \u05e9 \u05d3 \u05d2 \u05db \u05e2 \u05d9 \u05d7 \u05dc \u05da : ' {enter}",
-      "{shift} \u05d6 \u05e1 \u05d1 \u05d4 \u05e0 \u05de \u05e6 , . / {shift}",
-      ".com @ {space}",
+      " 1 2 3 4 5 6 7 8 9 0 - = {bksp}",
```

Comment:
There seems to be an empty key in the beginning of this first row. Is this intended? If you have a source please feel free to share!

Reply:
Yes, it's intended, I can't find a good source why this key should not output anything - but I guess you can add the Hebrew layout keyboard in some physical keyboard and see for yourself.

Some source I found (see the MacOs part):
https://www.muhlenberg.edu/media/contentassets/pdf/academics/llc/online-resources/hebrew/TypingGuides_printout_HBW_layout_mac_2pg.pdf

Some sources show the use of ';' instead of the empty key - if you prefer I'll add it.

For all the other keys: 
https://en.wikipedia.org/wiki/Hebrew_keyboard